### PR TITLE
Switch to ravel

### PR DESCRIPTION
**Pull request by thoradia**
This is a good start, but there is a long way to go

### CODE REVIEW

**Comment by thoradia** [comment]
The connman dbus signals should now be converted to ravel
I will move on to the connman dbus methods

**Comment by thoradia** [comment]
It is now possible again to connect a network device
I will stop here until I get feedback

**Comment by CvH** [comment]
@heitbaum @edit4ever @antonlacon  opinions ?

File: src/resources/lib/config.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import asyncio
```
Comment (CvH) [inline src/resources/lib/config.py:1]:
```suggestion
# SPDX-License-Identifier: GPL-2.0
# Copyright (C) 2020-present Team LibreELEC

import asyncio
```

header is missing

Comment (thoradia) [reply src/resources/lib/config.py:4]: Done

**Review by heitbaum — APPROVED** [review]

**Comment by heitbaum** [comment]
These are very similar to my tree (whilst mine is more on the BT)
Think the cleanup in conman is a great start.

**Comment by thoradia** [comment]
Than you for approving @heitbaum

**Comment by thoradia** [comment]
By the way, where is your tree?
I could not find it
Your findings would certainly help me further

**Pull request merged**